README.md

```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
 Create a `.commitlintrc.json` file and extend the default recommended config like so:
 ```js
 {
-  "extends": ["@biotope/quality-gate/config/.commitlintrc.js"]
+  "extends": ["@virtualidentityag/quality-gate/config/.commitlintrc.js"]
   // Add any other definition here.
 }
 ```
 
+## Recommended settings
+### Stencil
+#### .eslintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.eslintrc.js",
+  "rules": {
+    "class-methods-use-this": 0,
+    "no-script-url": 0,
+    "import/no-default-export": 1,
+    "react/prop-types": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "react/jsx-key": 0,
```

Comment:
why are we excluding key?

Reply:
This was the thing we experienced together where we would have had to set "keys" in arrays, but there weren't any arrays at all.